ui/src/pages/organizations/index.tsx

```diff
@@ -53,7 +61,7 @@ const Index = () => {
 
   return (
     <div className={c('list-container')}>
-      <Listing items={listItems} isFetching={isFetching} />
+      <Listing items={listItems} isFetching={isFetching} refetchData={changePagination} />
```

Comment:
Instead of `refetchData`, shouldn't we call it `onPaginate` here as well? It'd be more consistent, and will separate the responsibility of "re-fetching" and "paginating". Listing component shouldn't instruct its consumer to refetch new data, it should quietly show what data is given to it, and inform when user want to see next/previous page.